.changes/unreleased/tinymce-TINY-10741-2024-04-12-1.yaml

```diff
@@ -1,6 +1,6 @@
 project: tinymce
 kind: Fixed
-body: The buttons in the custom view header were clipped when overflowing
+body: The buttons in the custom view header were clipped on when overflowing.
```

Comment:
Adding the word `on` here seems incorrect?

Reply:
No problem. I can add it